QModManager/Patching/ModStatus.cs

```diff
@@ -1,10 +1,11 @@
-﻿namespace QModManager.API.ModLoading
+﻿namespace QModManager.Patching
 {
     internal enum ModStatus
     {
-        Success,
-        //CanceledByAuthor,
-        CanceledByUser,
+        Merged = -3,
+        //CanceledByAuthor = -2,
+        CanceledByUser = -1,
+        Success = 0,
```

Comment:
The remaining entries in this list should also be given explicit values

Reply:
Everything below Success will be automatically assigned a value > 0

Reply:
Be consistent